### ISSUE TITLE
Transaction pending

### ISSUE THREAD

**Issue by BalthazarG** [iotaledger/wallet#657]
Hi there !

I haven't touched my IOTAs for like 5 months and they have been on my wallet the whole time. I read that I had to upgrade my wallet and then replace my old seed with a new one using the reclaim tool (my IOTAs didn't appear in my wallet).

I tried doing this, several times, nothing worked. Then I tried loggin into both seeds several times and at somepoint, the IOTAs appeared in my old seed.
Tried using the reclaim tool once again but still didn't work.
I would now want to move my IOTAs but the transactions are always pending, nothing is happening.
https://thetangle.org/transaction/WLQJMTPYVMKYQZMVXJHIJGRMSMANRMTKCOQVJJMPFZKILRBJIBRSAYASMURPRZZBMPCSTDFWVQUQ99999

Does anyone know what is happening or how to make this transaction work ?

Thanks a lot !

**Comment by dingeldoof** [comment]
If your IOTAs didn't appear in your wallet, try reattaching your receiving address to the tangle and creating new addresses.

https://forum.helloiota.com/588/Help-My-IOTA-balance-is-zero-steps-to-help-you-find-your-balance-v252

**Comment by onemoreitguy** [comment]
Tried also that: now I got 3 address all with 0i with pending for more than 24 hours

**Comment by BalthazarG** [comment]
Hi, Thanks for your help guys.
As I said, the  correct amount of IOTAs is now appearing in my wallet. But the seed I use to see them is the one I had 6 months ago (no without the reclaim process etc). 
Reclaiming doesn't work. The transaction is pending and nothing happens even after a few days.

So I tried sending the coins from old seed (where the IOTAs appear) to a new one, nothing happened. So I then tried to send my IOTA's from my old seed to Bitfinex.
The transaction shows up and says -xxx IOTAs but the IOTAs are still in my wallet and the transaction is never confirmed. 

I tried joining the IOTA slack but I can't ;S
Don't really know what to try next

**Comment by BalthazarG** [comment]
Here is what I get when I try sending them 
![transaction unconfirmed](https://user-images.githubusercontent.com/16015420/33742639-9373e342-dba9-11e7-9353-3f437672eb9b.PNG)

**Comment by dingeldoof** [comment]
Yeah, the network is very bloated right now. It took me around 2 days to move my IOTA, reattaching every hour or so. Look for low ping nodes, keep at it. The private key reuse is normal.

**Comment by BalthazarG** [comment]
Thanks Dingeldoof ! I will keep trying.
Do you know with transaction I should try reattaching ? The first try I ever made or le last one ?
